src/main/java/com/cricket/project/service/impl/TeamServiceImpl.java

```diff
@@ -0,0 +1,117 @@
+package com.cricket.project.service.impl;
+
+import com.cricket.project.enums.PlayerType;
+import com.cricket.project.exception.TeamException;
+import com.cricket.project.model.Player;
+import com.cricket.project.model.Team;
+import com.cricket.project.repository.PlayerRepository;
+import com.cricket.project.repository.TeamRepository;
+import com.cricket.project.service.TeamService;
+import lombok.Data;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Service
+@Data
+public class TeamServiceImpl implements TeamService {
+
+    @Autowired
+    private PlayerRepository playerRepository;
+    @Autowired
+    private TeamRepository teamRepository;
+
+    private Map<Integer, Integer> exist = new HashMap<>();
```

Comment:
This can also cause problem, please find the problem and come up with a better solution.